src/main/java/com/rojel/wesv/WorldEditSelectionVisualizer.java

```diff
@@ -72,7 +72,12 @@ public boolean isHoldingSelectionItem(final Player player) {
                     return item.getType().getId() == wandItemField.getInt(WorldEdit.getInstance().getConfiguration());
                 } else if (wandItemField.getType() == String.class) {
                     final String itemTypeId = BukkitAdapter.adapt(player).getItemInHand(HandSide.MAIN_HAND).getType().getId();
-                    return itemTypeId.equals(wandItemField.get(WorldEdit.getInstance().getConfiguration()));
+                    if (itemTypeId.equals(wandItemField.get(WorldEdit.getInstance().getConfiguration()))) {
```

Comment:
At the moment, I cannot think of a good solution to solve this without compromising minecraft 1.8 compatibility.

The solutions I could think of:

# Combining both `if` statements using `||`
This would break 1.8 support, as the access to the OFF_HAND constant is read before the normal hand comparison is made.

# Delegating this to a seperate function

All this code if basically part of the main check, it really doesn't make sense to me to split it up.

# Suppressing the warning

Supressing warnings always looks ugly, and it's not up to me to decide that.